### PR TITLE
[Form] skip tests that do not work with ICU 71.1

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 7.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Comment by maxbeckers** [comment]
@xabbuh does this help you https://github.com/symfony/symfony/issues/52206#issuecomment-1776590581 ?

**Comment by xabbuh** [comment]
@maxbeckers Kind of, I suspect the ICU version used to be the cause here. At least that the only relevant difference that I can spot compared to 6.4 (see #52251).

Which version do you have locally? And can you also try different versions?

**Comment by maxbeckers** [comment]
@xabbuh you're right.

When i run it with ``70.1``, it works. When I use ``71.1``, it fails.

**Comment by xabbuh** [comment]
73.2 works for me too.

**Comment by maxbeckers** [comment]
I currently only have the two versions I tried for testing.

File: src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php [hunk 1]
```diff
@@ -105,6 +107,17 @@ public function testSubmitFromSingleTextDateTime()
...
+        if (!$form->isValid()) {
+            var_dump(Intl::getIcuVersion());
+            /** @var FormError $error */
+            foreach ($form->getErrors(true) as $error) {
+                var_dump($error->getMessage());
+                var_dump($error->getCause()->getMessage());
+                var_dump($error->getCause()->getInvalidMessage());
+                var_dump($error->getCause()->getInvalidMessageParameters());
+            }
+        }
```
Comment (maxbeckers) [inline src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php]: The output of this looks like that if it helps:
```
string(4) "71.1"
string(26) "Please enter a valid date."
string(84) "Unable to reverse value for property path "date": Date parsing failed: U_PARSE_ERROR"
NULL
array(0) {
}
```

**Pull request merged**